app/lib/assessment_factory.rb

```diff
@@ -221,7 +221,7 @@ def work_history_section
       if application_form.created_under_new_regulations?
         %i[verify_school_details work_history_references]
       else
-        %i[email_contact_current_employer satisfactory_evidence_work_history]
+        %i[email_contact_current_employer satisfactory_evidence_work_history work_history_references]
```

Comment:
Do we definitely need this change? Under the old regulations we don't send out references.

Reply:
Yeah i dont think we need it; i think there was a test failure here without it. Will investigate.